app/config/BackendConfig.scala

```diff
@@ -47,4 +47,10 @@ class BackendConfig @Inject()(val servicesConfig: ServicesConfig,
 
   lazy val dailyQuota: Int = servicesConfig.getConfInt("constants.daily-quota", 1)
 
+  lazy val nonRepudiationSubmissionUrl: String = servicesConfig.baseUrl("non-repudiation") + "/submission"
+  lazy val nonRepudiationApiKey: String = servicesConfig.getConfString(
```

Comment:
`servicesConfig.getString` throws a runtime exception if it fails to find the config and dumps the config key in to the exception message so you don't have to manually specify it.